src/components/CreateInstructorPage/index.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+
+import { CreateInstructorForm } from './CreateInstructorForm';
+import StatusAlert from '../StatusAlert';
+import LoadingSpinner from '../LoadingSpinner';
+
+class CreateInstructorPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      startedFetching: false,
+    };
+    this.handleInstructorCreate = this.handleInstructorCreate.bind(this);
+    this.setStartedFetching = this.setStartedFetching.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.fetchInstructorOptions();
+    this.setStartedFetching();
+  }
+
+  setStartedFetching() {
+    this.setState({ startedFetching: true });
+  }
+
+  processSocialLinks(links) {
+    if (!links) {
+      return [];
+    }
+
+    return links.map(link => (
+      ({
+        id: null,
+        type: link.type,
+        title: link.title,
+        url: link.url,
+      })
```

Comment:
This could maybe also be written like `Object.assign({}, link, {id: null})` or hell `Object.assign({id: null}, link)` - but then you're getting all fields of link instead of just 3, if you're worried about surprise stowaway fields.

Reply:
I really look forward to standardizing on es7, so we can just do `{...link, id: null}` without the `Object.assign` boilerplate.